src/api/index.js

```diff
@@ -52,6 +52,7 @@ import getUsers from './users/getUsers';
 import createUser from './users/createUser';
 import getUserProfile from './users/getUserProfile';
 import updateUserStatus from './users/updateUserStatus';
+import getTemporaryFileUrl from './messages/getTemporaryFileUrl';
```

Comment:
6b70e6eca api [nfc]: Create endpoint to get temporary URL to access uploaded file.

In the commit message:

Let's remove `[nfc]`; quoting @gnprice from https://github.com/zulip/zulip-mobile/pull/4089#discussion_r421215261,

> I agree I wouldn't label them `[nfc]`, for the reason you mention at the end.

in response to me,

> In person, I think, @gnprice once told me that a commit that adds completely new code shouldn't usually be marked `[nfc]`, even when it's not used. That's why, e.g., 6d6dec1 and b41554b aren't marked that way.

Reply:
Also in the commit message, let's add the same details you added in the jsdoc, in response to https://github.com/zulip/zulip-mobile/pull/4089#discussion_r421883935, where I quoted Greg from [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/view.20uploaded.20files/near/819779):

> The goal basically is to get
> * (a) a plain old URL that can be passed to a browser (not a URL plus some cookies or other headers that need to be set)
> * (b) that gets the user the file immediately, without requiring a fresh login
> * (c) but doesn't include a secret with much wider powers (i.e. their API key), which would risk a nasty surprise if they innocently show someone else the file by sending them the link.

I think a bulleted or numbered list like this is a good, readable format; I think you could even use this quote directly, if you want, with proper attribution to Greg.

Reply:
Done and Done.